examples/step-17/doc/results.dox

```diff
@@ -9,15 +9,15 @@ most basic way to run MPI programs is using a command line like
 @code
   mpirun -np 32 ./step-17
 @endcode
-to run the step-17 executable with 32 processors.
+to run the step-17 executable with 32 processors. 
 
-The command line above is the appropriate way of starting the program
-on a multicore machine when using MPI for parallelization. On the
-other hand, most clusters are shared resources and have some kind of
-scheduling system that distributes jobs onto available processors. All
-of these scheduling systems have their own calling syntax - on my system, I have to use the command
-<code>qsub</code> with a whole host of options to run a job in parallel - so
-that the exact command line syntax varies.
+(If you work on a cluster,
+there is typically a step in between where you need to set up a job script,
+submit the script to a scheduler that then executes the script whenever it
```

Comment:
I had some trouble reading this sentence; the third clause is missing something. How about

```
If you work on a cluster, then there is typically a step in between where you
need to set up a job script and submit the script to a scheduler. The scheduler
will execute the script whenever it can allocate 32 unused processors for your
job.
```
instead?